src/hotspot/share/classfile/placeholders.cpp

```diff
@@ -120,17 +124,16 @@ Symbol* PlaceholderTable::find_entry(int index, unsigned int hash,
   // If entry exists, reuse entry
   // For both, push SeenThread for classloadAction
   // if havesupername: this is used for circularity for instanceklass loading
-PlaceholderEntry* PlaceholderTable::find_and_add(int index, unsigned int hash,
+PlaceholderEntry* PlaceholderTable::find_and_add(unsigned int hash,
                                                  Symbol* name,
                                                  ClassLoaderData* loader_data,
                                                  classloadAction action,
                                                  Symbol* supername,
                                                  Thread* thread) {
-  PlaceholderEntry* probe = get_entry(index, hash, name, loader_data);
+  PlaceholderEntry* probe = get_entry(hash, name, loader_data);
```

Comment:
Minor nit of a comment since you are in this file and specifically in this method can you consider changing the if statement conditional at line #143 to explicitly check if (probe != NULL)?  I noticed line #167 has this type of conditional and it would be good to have consistency.  I think there are some other cases in this file, like line #119, where it is inconsistent.

Reply:
The if (probe) conditional on line #143 isn't necessary because we've just created a placeholder and code above would have null pointer references if it failed (in add_entry -> new_entry), so I removed it.  I fixed another check against NULL just above this. I don't see any others here at least.
Thanks for reviewing!